Blimp/Blimp.h

```diff
@@ -416,8 +416,6 @@ class Blimp : public AP_Vehicle
     void read_rangefinder(void);
     bool rangefinder_alt_ok();
     bool rangefinder_up_ok();
-    void rpm_update();
-    void update_optical_flow(void);
```

Comment:
Unintentional remove here? `void update_optical_flow(void);`

Reply:
Nope, just trying to sneak it in.  You'll note the commit message is generic :-)

Reply:
I guess we will let you have it ;)